cmd/snap/cmd_snap_op.go

```diff
@@ -144,15 +144,62 @@ func (x *cmdRemove) Execute([]string) error {
 	return nil
 }
 
+type channelMixin struct {
+	Channel string `long:"channel" description:"Use this channel instead of the device's default"`
```

Comment:
I think this should say "instead of stable"  (there's no device's default anymore)